src/components/NotesPage/NotesPageComponent.js

```diff
@@ -1,32 +1,24 @@
 import React, { Component } from 'react';
 import PropTypes from 'prop-types';
 
-import { ControlsHelpComponent } from 'components/ControlsHelp/ControlsHelpComponent';
-import { NotesContentEditorContainer } from 'components/NotesContentEditor/NotesContentEditorContainer';
-import { NotesMindMapContainer } from 'components/NotesMindMap/NotesMindMapContainer';
-import { LogoutButtonContainer } from 'components/LogoutButton/LogoutButtonContainer';
-import { SearchPanelContainer } from 'components/SearchPanel/SearchPanelContainer';
-import { TopBarComponent } from 'components/TopBar/TopBarComponent';
+// import { ControlsHelpComponent } from 'components/ControlsHelp/ControlsHelpComponent';
+import { NotesMindMapContainer } from 'components/NotesMindMap/NotesMindMapContainer'; // Notes tree view
+import { TopBar } from 'components/TopBar/TopBar';
 import { UploadStatusButton } from 'components/Uploads/UploadStatusButton';
-import { UploadButton } from 'components/Uploads/UploadButton';
-import { OpenFileButtonComponent } from 'components/OpenFileButton/OpenFileButtonComponent';
+// import { UploadButton } from 'components/Uploads/UploadButton';
 import BottomBarComponent from 'components/BottomBar/BottomBarComponent';
 
 export class NotesPageComponent extends Component {
   render() {
-    const { isHelpViewed, closeHelp } = this.props;
+    // const { isHelpViewed, closeHelp } = this.props;
 
     return (
       <>
-        <TopBarComponent />
-        <OpenFileButtonComponent />
+        <TopBar />
         <NotesMindMapContainer />
-        <NotesContentEditorContainer />
-        {!isHelpViewed && <ControlsHelpComponent onClose={closeHelp} />}
-        <SearchPanelContainer />
-        <LogoutButtonContainer />
+        {/* {!isHelpViewed && <ControlsHelpComponent onClose={closeHelp} />} */}
         <UploadStatusButton />
-        <UploadButton />
+        {/* <UploadButton /> */}
```

Comment:
Why did you remove upload button?

Reply:
Upload button functionality are delegated to new design fab button. Please checkout to branch and run app to check application functionality